### PR TITLE
Components - Milestone 2

### PR DESCRIPTION
**Pull request by SevlaMare**
### Create Components:
- [x] App
- [x] Display
- [x] Button
- [x] Button Panel

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <h1>Hi from app</h1>
+
+      <Display />
```
Comment (whiz25) [inline src/components/App.js]: - You don't need `<h1>Hi from app</h1>` so remove it. Wrap the `<Display />` component in the `<div>` component.
```html
<div>
    <Display />
    <ButtonPanel />
</div>
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray: 
You have done well. You are almost there. Let's fix something small so we can merge and proceed to the next project. Check my comment for suggestion.


### Martin Afani

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <Display />
+
+      <div className="btnPanel">
+        <ButtonPanel />
+      </div>
```
Comment (suretrust) [inline src/components/App.js:12]: For clarity and modularity, you could move this to the `ButtonPanel` component.
```suggestion
      <ButtonPanel />
```

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  'AC', '+/-', '%', '÷',
+  '7', '8', '9', 'X',
+  '4', '5', '6', '-',
+  '1', '2', '3', '+',
+  '0', '.', '=',
+];
+
+const ButtonPanel = () => (
+  buttons.map(element => <Button key={element.id} buttonName={element.toString()} />)
+);
```
Comment (suretrust) [inline src/components/ButtonPanel.js:14]: So this becomes something like:
```suggestion
const ButtonPanel = () => {
  return (
    <div className="btnPanel">
      buttons.map(element =>
        <Button key={element.id} buttonName={element.toString()} />)
    </div>
  )
};
```

**Review by suretrust — APPROVED** [review]
Hi @SevlaMare, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

**Note:** I added an extra suggestion below but it's entirely optional.

_Good luck with your future milestones and happy coding!_ 🎉

File: src/components/App.js (hunk 2 above)
Comment (SevlaMare) [reply src/components/App.js:12]: Thanks, it's implemented on milestone 3.

**Pull request merged**